style_admin.css

```diff
@@ -157,7 +132,7 @@ a {
 
 /* Style for login box */
 .login_box {
-	background: #fff;
+	background: #121416;

```

Comment:
Majority consists of applying changing the background colour and font family from here and below.